docs/user/admin/settings.rst

```diff
@@ -824,3 +824,55 @@ To Re-enable Data Sources:::
       }
     }
 
+plugins.query.field_type_tolerance
```

Comment:
Add to `Table of contents`

Reply:
I see `plugins.query.field_type_tolerance` in the table of contents.
https://github.com/Bit-Quill/opensearch-project-sql/blob/issue-1300-documentation/docs/user/admin/settings.rst